ERCS/erc-7566.md

```diff
@@ -0,0 +1,193 @@
+---
+eip: 7566
+title: Multiplayer Onchain Game
+description: A solution that enables multiple players to play onchain game.
```

Comment:
Your description doesn't give any new information that wasn't already in the title. You should also try to avoid words that don't add substance, like "solution".